pkg/repository/repository.go

```diff
@@ -8,6 +8,20 @@ import (
 	"github.com/harness/ff-golang-server-sdk/storage"
 )
 
+// Repository interface for data providers
+type Repository interface {
```

Comment:
Do we have any other repositories at the moment?

Reply:
No, we will have always one repository with configurable cache and storage